sympy/polys/polytools.py

```diff
@@ -6231,11 +6231,11 @@ def sqf_list(f, *gens, **args):
         except PolificationFailed:
             coeff *= b
         else:
-            gen_set.add(poly.gens)
+            for gen in poly.gens:
+                gen_set.add(gen)
```

Comment:
How about `gen_set.update(gens)`?

Reply:
yes, update function would also do the thing